### PR TITLE
automate OCP-30193 (for stage image test)

### PR DESCRIPTION
**Pull request by weinliu**

### CODE REVIEW

**Comment by weinliu** [comment]
@jhou1 @sunilcio @lyman9966 Would you mind taking a look? log :https://privatebin-it-iso.int.open.paas.redhat.com/?ba5e4e08d8a9bbf3#iavPRPG6AxznaB7KGAeagdEEC0Sun+501gQRRKRxpdA=

File: features/step_definitions/admin.rb [hunk 1]
```diff
@@ -12,9 +12,10 @@
     f.write(@result[:response])
     f.close
     # secret name `admin-kubeconfig` must match template of e2e tests
-    @result = admin.cli_exec(:secret_new,
-                             secret_name: "admin-kubeconfig",
-                             source: "admin.kubeconfig=#{f.path}",
+    @result = admin.cli_exec(:create_secret,
+			     secret_type: "generic",
```
Comment (jhou1) [inline features/step_definitions/admin.rb:16]: Align the indention here.

File: features/step_definitions/helper_services.rb [hunk 2]
```diff
@@ -708,7 +708,10 @@
   })
   step 'the step should succeed'
   # cluster-capacity as a target pod
-  step 'I run oc create over ERB test file: infrastructure/cluster-capacity/cluster-capacity-pod.yaml'
+  step %Q/I run the :create client command with:/, table(%{
+    | f         | #{ENV['BUSHSLICER_HOME']}/testdata/infrastructure/cluster-capacity/cluster-capacity-pod.yaml |
+    | namespace | #{project.name} |
```
Comment (jhou1) [inline features/step_definitions/helper_services.rb:713]: Align the table here.

File: lib/openshift/horizontal_pod_autoscaler.rb [hunk 3]
```diff
@@ -21,5 +21,29 @@ def current_cpu_utilization_percentage(user: nil, cached: true, quiet: false)
...
+#      return obj.dig('spec', 'targetAverageValue')
+#    end
+
+#    def current_average_value(user: nil, cached: true, quiet: false)
+#      raw_resource(user: user, cached: cached, quiet: quiet).
+#        dig('status', 'currentAverageValue').
+#        to_i # it can be nil when zero thus using #to_i
+#    end
+#    
+#
```
Comment (jhou1) [inline lib/openshift/horizontal_pod_autoscaler.rb:36]: Can you please remove those lines of commented code?

File: lib/openshift/horizontal_pod_autoscaler.rb [hunk 4]
```diff
@@ -21,5 +21,29 @@ def current_cpu_utilization_percentage(user: nil, cached: true, quiet: false)
...
+      obj = raw_resource(user: user, cached: cached, quiet: quiet)
+      return obj.dig('spec', 'targetAverageUtilization')
+    end
+
+    def current_average_utilization(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).
+        dig('status', 'currentAverageUtilization').
+        to_i # it can be nil when zero thus using #to_i
+    end
+
```
Comment (jhou1) [inline lib/openshift/horizontal_pod_autoscaler.rb:47]: The two methods: `target_average_utilization` returns a string or nil, `current_average_utilization` returns integer. Judging from their names, they should return the same data type. Can you let `target_average_utilization` return an integer as well?

**Comment by weinliu** [comment]
The stage script was supposed to placed in cucushift, and also I made some updates on the case to make the testing more suitable. Closing this PR, and tracking the progress with https://github.com/openshift/cucushift/pull/7727

**Pull request closed without merging**